databook/background/neuropixels-description.md

```diff
@@ -2,11 +2,11 @@
 
 # TO DO: NEEDS FIGURES
 
-Neuropixels are silicon probes which continuously detect voltage fluctuations in the surrounding neural tissue on 374 or 384 channels (depending on the model). Each channel is split into two separate data streams, or *bands*, on the probes. The *spike band* is digitized at 30 kHz with a 500 Hz high-pass filter, and contains information about action potentials fired by neurons directly adjacent to the probe. The *LFP band* is digitized at 2.5 kHz, and records the low-frequency (<1000 Hz) fluctuations that result from synchronized neural activity over a wider area. The shanks are generally 10 mm long shank with a 70 µm width.
+Neuropixels are silicon probes which continuously detect voltage fluctuations in the surrounding neural tissue on 374 or 384 channels (depending on the model). Each channel is split into two separate data streams, or *bands*, on the probes. The *spike band* is digitized at 30 kHz with a 500 Hz high-pass filter, and contains information about action potentials fired by neurons directly adjacent to the probe. The *LFP band* is digitized at 2.5 kHz, and records the low-frequency (<1000 Hz) fluctuations that result from synchronized neural activity over a wider area. The shanks are generally 10 mm long with a 70 µm width.
```

Comment:
might be good to add links to the glossary for action potentials and LFP

Reply:
revised